<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deletelogin</class>
 <widget class="QWidget" name="deletelogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="login_IDlineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_deletename_lineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deletequeryButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>登入密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dele_login_passwordlineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="login_deleteButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletebackButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletecancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>删除用户登入信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
